packages/core/contracts/cross-chain-oracle/chain-adapters/Admin_ChildMessenger.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "../interfaces/ChildMessengerInterface.sol";
+import "../interfaces/ChildMessengerConsumerInterface.sol";
+import "../../common/implementation/Lockable.sol";
+
+/**
+ * @notice A version of the child messenger that allows an admin to relay messages on its behalf.
+ * @dev No parent messenger is needed for this case, as the admin could be trusted to manually send DVM requests on
+ * mainnet. This is intended to be used as a "beta" deployment compatible with any EVM-compatible chains before
+ * impleenting a full bridge adapter. Put simply, it is meant as a stop-gap.
+ */
+contract Admin_ChildMessenger is Ownable, Lockable, ChildMessengerInterface {
+    // The only child network contract that can send messages over the bridge via the messenger is the oracle spoke.
+    address public oracleSpoke;
+
+    event SetOracleSpoke(address newOracleSpoke);
+    event MessageSentToParent(bytes data, address indexed oracleSpoke);
```

Comment:
Should we emit `oracleSpoke` in the other `ChildMessenger` contracts events? Currently we do not but I'm curious if you think its useful and you could add in this PR

Reply:
The only one where it's currently missing is `MessageReceivedFromParent`, right?

I would say no for that one only because it seems somewhat irrelevant. If the target is the oracle spoke, that address will be in the log, but if you're talking to the governors spoke (or any other contract), why do you care about the oracle spoke?

Note: not strongly opinionated, just seems a little irrelevant for that particular call.

Thoughts?

Reply:
No I'm talking about the `MessageSentToParent` events in teh `Optimstic/Polygon/Arbitrum_ChildMessenger`

Reply:
Added.